test/expected/edition.out

```diff
@@ -14,40 +14,34 @@ SELECT _timescaledb_internal.tsl_loaded();
  t
 (1 row)
 
-SELECT _timescaledb_internal.enterprise_enabled();
- enterprise_enabled 
---------------------
- f
-(1 row)
-
 \unset ECHO
-SET timescaledb.license_key='ApacheOnly';
-ERROR:  invalid value for parameter "timescaledb.license_key": "ApacheOnly"
-DETAIL:  Cannot downgrade a running session to Apache Only.
+SET timescaledb.license='apache';
+ERROR:  invalid value for parameter "timescaledb.license": "apache"
```

Comment:
misleading error since "apache" is a valid parameter value. Just not allowed to be set in this way. Is it possible to have a more generic error, e.g., "cannot set TimescaleDB license" and then let the detail message tell us whether the parameter values is not allowed or it is not possible to downgrade.

Reply:
I think we can add them in the same way was done for other guc hints. Just to note, that guc hooks should not throw errors

Reply:
Actually I am not sure we can change this. Seems like the best we can do here, is to show DETAIL and HINT, which we already did

Reply:
So, is "invalid value for parameter..." a PostgreSQL error? I guess we'll just have to live with this in that case.

Reply:
Yeah, unfortunately. At least we have additional details with it